releases/point-2022-3.md

```diff
@@ -0,0 +1,81 @@
+
+See detailed step-by-step instructions on how to upgrade SysKit Point in the [upgrade article](../installation/upgrade-syskit-point.md).
+
+## Features
+* **SysKit Point provisioning templates allow choosing metadata columns** to include additional attributes when generating reports.
+    * The metadata columns available in the dropdown menu provide:
+        * **Setting custom values**
+        * **Users can define and fill in the value when creating a request**
+        * **The ability to create and edit new metadata columns**
+    * **Admins can define values** for metadata columns for every site, team or group on the Overview screen.
+        *The pre-defined metadata **set by the admin will be shown to the user as read-only**.
+    * For more information on **SysKit Point metadata, check out this article**.
+* **Service accounts can be added to an ignore list**.
+    * Setting up an ignored service account means:
+        * The Service Account page hits and views **will be ignored when detecting any workspace activity**.
+        * The Service Account audit logs **won’t be stored**.
+        * The Service Account changes **will be ignored and won’t trigger alerts**.
+    * Read more about **how to Ignore Service accounts here**.
+* **Added the ability to remove Sharing links** when generating the report.
+    * Select All feature available for removing all links attached to a certain Site.
+* **Configuration Inventory Module (CIM) settings have been added to SysKit Point**.
+    * Additional feature added that allows selecting who receives the CIM email.
+
+
+## Improvements & Bug Fixes
+
+* **Emails sent by SysKit Point now have customized dynamic subject field**.
+     * The subject line shows the context of the email; for example, Communication site is inactive.
+* **Improved the loading speed** for Access Review Options, **improved the functionality** of the preview email and dialogue on **the Access Review Options page**.
+* **Improved performance** for Privileged Identity Management (PIM).
+    * Privileged Identity Management (PIM) allows some users to assign themselves Global Admin rights for a limited amount of time.
+    * Global Admin sync improved to occurs every 4 minutes. 
+    * Fixed a bug where AAD groups weren’t taken into account for PMI if some members of those groups had administrative roles assigned.
+* **Improved speed and performance** for the One Drive tile when accessing through User details.
+* **Optimized speed and performance** when loading the Lifecycle Management details screen.
+* **Improved speed** for loading the Overview tile when accessing the Private Channel details screen.
+* **Improved messaging** when a Bad Request or Internal server error occurs that provides more insight into the issue that occurred when more information is available.
```

Comment:
Ovo bih čak maknuo. Na kraj RN-a staviti 
* **Various UX/UI improvements and minor fixes** are available.